### PR TITLE
[zh-cn] update the translation of "Adding vector graphics to the web"

### PR DESCRIPTION
**Pull request by hanyujie2002**
~Waiting for upstream https://github.com/mdn/content/pull/33184 and https://github.com/mdn/content/pull/33185~

Relates to #18313

### CODE REVIEW

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 1]
```diff
@@ -1,60 +1,61 @@
 ---
-title: 在网页中添加矢量图形
+title: 向 web 中添加矢量图形
 slug: Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web
+l10n:
+  sourceCommit: a3d9f61a8990ba7b53bda9748d1f26a9e9810b18
```
Comment (hanyujie2002) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:5]:
```suggestion
  sourceCommit: 2492742db9d7341fa74604a1b4cd97dc3c079cab
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
-<table class="learn-box standard-table">
+<table>
   <tbody>
     <tr>
       <th scope="row">前提：</th>
       <td>
-        你需要了解
-        <a href="/zh-CN/docs/Learn/HTML/Introduction_to_HTML"
+        你需要了解<a
+        href="/zh-CN/docs/Learn/HTML/Introduction_to_HTML"
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:18]: HTML 不是英文单词吗，为啥要把这里的空格去掉呢（去掉了换行）

```suggestion
        你需要了解
        <a href="/zh-CN/docs/Learn/HTML/Introduction_to_HTML"
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 3]
```diff
@@ -1,60 +1,61 @@
...
+        >并且知道如何<a
+         href="/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML"
+          >在文档中插入图片</a
+        >。
       </td>
     </tr>
     <tr>
       <th scope="row">目标：</th>
-      <td>了解如何嵌入 SVG (矢量) 图形到网页中。</td>
+      <td>了解如何在网页中嵌入 SVG (矢量) 图形。</td>
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:28]:
```suggestion
      <td>了解如何在网页中嵌入 SVG（矢量）图形。</td>
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 4]
```diff
@@ -1,60 +1,61 @@
...
   </tbody>
 </table>
 
-> **备注：** 本文的目的并不是教你 SVG；仅仅是告诉你它是什么，以及如何在网页中添加它。
+> **备注：** 本文的目的并不是教你 SVG；仅仅是告诉你它是什么，以及如何在 web 页面中添加它。
 
 ## 什么是矢量图形？
 
-在网上，你会和两种类型的图片打交道 — 位图和矢量图：
+在 web 上，你会和两种类型的图片打交道——**位图（raster image）**和**矢量图**：
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:37]: 位图是很常见的名词了，没必要在加注释

```suggestion
在 web 上，你会和两种类型的图片打交道——**位图**和**矢量图**：
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 5]
```diff
@@ -1,60 +1,61 @@
...
 
-- 位图使用像素网格来定义 — 一个位图文件精确得包含了每个像素的位置和它的色彩信息。流行的位图格式包括 Bitmap (`.bmp`), PNG (`.png`), JPEG (`.jpg`), and GIF (`.gif`.)
-- 矢量图使用算法来定义 — 一个矢量图文件包含了图形和路径的定义，电脑可以根据这些定义计算出当它们在屏幕上渲染时应该呈现的样子。 {{glossary("SVG")}} 格式可以让我们创造用于 Web 的精彩的矢量图形。
+- **位图**使用像素网格来定义——位图文件精确包含每个像素的位置和它的色彩信息。流行的 web 位图格式包括 Bitmap（`.bmp`）、PNG（`.png`）、JPEG（`.jpg`），以及 GIF (`.gif`)。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:39]:
```suggestion
- **位图**使用像素网格来定义——位图文件精确包含每个像素的位置和它的色彩信息。流行的 web 位图格式包括 Bitmap（`.bmp`）、PNG（`.png`）、JPEG（`.jpg`），以及 GIF（`.gif`）。
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 6]
```diff
@@ -72,53 +73,53 @@ slug: Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web
 
 {{ EmbedLiveSample('SVG 是什么？', 300, 200) }}
 
-从上面的例子可以看出，SVG 很容易手工编码。是的，你可以在文本编辑器中手动编写简单的 SVG，但是对于复杂的图像，这很快就开始变得非常困难。为了创建 SVG 图像，大多数人使用矢量图形编辑器，如 [Inkscape](https://inkscape.org/en/) 或 [Illustrator](https://en.wikipedia.org/wiki/Adobe_Illustrator)。这些软件包允许你使用各种图形工具创建各种插图，并创建照片的近似值（例如 Inkscape 的跟踪位图功能）。
+从上面的例子可以看出，SVG 很容易手工编码。是的，你可以在文本编辑器中手动编写简单的 SVG，但是随着图像复杂度提升，手动编码很快就开始变得非常困难。为了创建 SVG 图像，大多数人使用矢量图形编辑器，如 [Inkscape](https://inkscape.org/en/) 或 [Illustrator](https://zh.wikipedia.org/zh-cn/Adobe_Illustrator)。借助这些软件包，你可以使用各种图形工具创建插图，也可以创建近似照片的矢量图（例如 Inkscape 的跟踪位图功能）。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:76]:
```suggestion
从上面的例子可以看出，SVG 很容易手工编码。是的，你可以在文本编辑器中手动编写简单的 SVG，但是随着图像复杂度提升，手动编码很快就开始变得非常困难。为了创建 SVG 图像，大多数人使用矢量图形编辑器，如 [Inkscape](https://inkscape.org/) 或 [Illustrator](https://zh.wikipedia.org/wiki/Adobe_Illustrator)。借助这些软件包，你可以使用各种图形工具创建插图，也可以创建近似照片的矢量图（例如 Inkscape 的跟踪位图功能）。
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 7]
```diff
@@ -72,53 +73,53 @@ slug: Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web
...
+### 快捷方式：`img` 元素
 
-要通过 {{htmlelement("img")}}元素嵌入 SVG，你只需要按照预期的方式在 src 属性中引用它。你将需要一个`height`或`width`属性（或者如果你的 SVG 没有固有的宽高比）。如果你还没使用过\<img>元素，请阅读[HTML 中的图片](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML)教程。
+要通过 {{htmlelement("img")}} 元素嵌入 SVG，你只需要按照预期的方式在 src 属性中引用它。你将至少需要 `height` 或 `width` 属性中的一个（或者如果你的 SVG 没有固有的宽高比，则都需要）。如果你还没使用过 `img` 元素，请阅读 [HTML 中的图片](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML)教程。
 
 ```html
 <img
   src="equilateral.svg"
-  alt="triangle with all three sides equal"
+  alt="三条边相同的三角形"
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:103]:
```suggestion
  alt="等边三角形"
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 8]
```diff
@@ -127,21 +128,21 @@ SVG 除了迄今为止所描述的以外还有其他优点：
...
 background: url("fallback.png") no-repeat center;
 background-image: url("image.svg");
 background-size: contain;
 ```
 
-像上面描述的`<img>`方法一样，使用 CSS 背景图像插入 SVG 意味着它不能被 JavaScript 操作，并且也受到相同的 CSS 限制。
+与上文所述的 `<img>` 方法一样，使用 CSS 背景图片插入 SVG 意味着 SVG 无法使用 JavaScript 进行操作，并且同样受到相同的 CSS 限制。
 
-如果 SVG 根本没有显示，可能是因为你的服务器设置不正确。如果是这个问题，[这篇文章](/zh-CN/docs/Web/SVG/Tutorial/Getting_Started#A_Word_on_Webservers)将告诉你正确方向。
+如果 SVG 根本没有显示，可能是因为你的服务器设置不正确。如果是这个问题，[这篇文章]将为你指出正确方向 (/zh-CN/docs/Web/SVG/Tutorial/Getting_Started#A_Word_on_Webservers)。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:141]:
```suggestion
如果 SVG 根本没有显示，可能是因为你的服务器设置不正确。如果是这个问题，[这篇文章](/zh-CN/docs/Web/SVG/Tutorial/Getting_Started#关于_web_服务器上_.svgz_文件的小提示)将为你指出正确方向。
```

**Review by yin1999 — COMMENTED** [review]
一些建议：

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 9]
```diff
@@ -179,24 +173,22 @@ background-size: contain;
 
 #### 缺点
 
-- 如你所知， `iframe`有一个回退机制，如果浏览器不支持`iframe`，则只会显示回退。
+- 如你所知，`iframe` 有一个回退机制，如果浏览器不支持 `iframe`，则只会显示回退。
 - 此外，除非 SVG 和你当前的网页具有相同的 {{glossary('origin')}}，否则你不能在主页面上使用 JavaScript 来操纵 SVG。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:177]:
```suggestion
- 此外，除非 SVG 和你当前的网页具有相同的{{glossary('origin', '来源')}}，否则你不能在主页面上使用 JavaScript 来操纵 SVG。
```

File: files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md [hunk 10]
```diff
@@ -179,24 +173,22 @@ background-size: contain;
...
 
 ## 动手学习：使用 SVG
 
-在这个动手学习部分中，我们希望你能够体验一下 SVG 的乐趣。在下面的“input”部分，你将看到我们已经提供了一些样例来开始使用。你还可以访问 [SVG 元素参考](/zh-CN/docs/Web/SVG/Element)，了解更多关于 SVG 可以使用的其他玩具的细节，也可以尝试一下。这部分都是为了锻炼你的研究技巧，并且有一些乐趣。
+在本动手学习部分中，我们希望你能够体验一下 SVG 的乐趣。在下面的 _input_ 部分，你会看到我们已经提供了一些样例供你使用。你还可以访问 [SVG 元素参考](/zh-CN/docs/Web/SVG/Element)，了解更多 SVG 可以把玩的细节，当然也可以尝试一下。本部分都是为了锻炼你的研究技巧，并且有一些乐趣。
```
Comment (yin1999) [inline files/zh-cn/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.md:181]:
```suggestion
在本动手学习部分中，我们希望你能够体验一下 SVG 的乐趣。在下面的*输入*部分，你会看到我们已经提供了一些样例供你使用。你还可以访问 [SVG 元素参考](/zh-CN/docs/Web/SVG/Element)，了解更多 SVG 可以把玩的细节，当然也可以尝试一下。本部分都是为了锻炼你的研究技巧，并且有一些乐趣。
```

**Review by yin1999 — APPROVED** [review]
Thanks @hanyujie2002 :+1:

**Pull request merged**